README.md

```diff
@@ -111,5 +128,5 @@ if(ctx.numeric_labels != null && ctx.numeric_labels.elastic_span_self_time != nu
 
 # License
 
-The Elastic Distribution for OpenTelemetry Java is licensed under [Apache License, Version 2.0](https://www.apache.org/licenses/LICENSE-2.0.html).
```

Comment:
I also prefer the "of" instead of "for". I think this exact wording comes from the OTel spec (however the "for" still feels weird to me but I can accept it as a non-native english speaker). If we keep pushing this change then there might be other places to update for consistency too.